arclight-common/src/main/java/io/izzel/arclight/common/mixin/core/world/WorldMixin.java

```diff
@@ -57,6 +57,11 @@
     @Shadow public abstract WorldBorder getWorldBorder();
     @Shadow@Final private WorldBorder worldBorder;
     @Shadow public abstract long getDayTime();
+
+    @Shadow public abstract Chunk getChunkAt(BlockPos pos);
```

Comment:
Please follow the code format. If your formatter does not work well, format it manually.